runtime/centrifuge/src/migrations.rs

```diff
@@ -10,45 +10,6 @@
 // MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 // GNU General Public License for more details.
 
-use cfg_primitives::AccountId;
-use frame_support::parameter_types;
-use sp_std::{vec, vec::Vec};
-
-use crate::{OraclePriceCollection, OraclePriceFeed};
-
-// Number of identities on Centrifuge Chain on 29.05.2024 was 34
-const IDENTITY_MIGRATION_KEY_LIMIT: u64 = 1000;
-
-parameter_types! {
-	// Address used by Anemoy to withdraw in AssetHub
-	// 4dTeMxuPJCK7zQGhFcgCivSJqBs9Wo2SuMSQeYCCuVJ9xrE2 --> 5CSfibAR9HHEwKpXbdsDHguAmLgKS6yKat2joTLA75DZ152L
-	pub AccountMap: Vec<(AccountId, AccountId)> = vec![
-		(
-			AccountId::new(hex_literal::hex!("5dbb2cec05b6bda775f7945827b887b0e7b5245eae8b4ef266c60820c9377185")),
-			AccountId::new(hex_literal::hex!("10c03288a534d77418e3c19e745dfbc952423e179e1e3baa89e287092fc7802f"))
-		)
-	];
-}
-
 /// The migration set for Centrifuge @ Polkadot.
 /// It includes all the migrations that have to be applied on that chain.
-pub type UpgradeCentrifuge1103 = (
-	runtime_common::migrations::increase_storage_version::Migration<OraclePriceFeed, 0, 1>,
-	runtime_common::migrations::increase_storage_version::Migration<OraclePriceCollection, 0, 1>,
-	pallet_collator_selection::migration::v1::MigrateToV1<crate::Runtime>,
-	pallet_collator_selection::migration::v2::MigrationToV2<crate::Runtime>,
-	runtime_common::migrations::loans::AddWithLinearPricing<crate::Runtime>,
-	runtime_common::migrations::hold_reason::MigrateTransferAllowListHolds<
-		crate::Runtime,
-		crate::RuntimeHoldReason,
-	>,
-	// Migrations below this comment originate from Polkadot SDK
-	pallet_xcm::migration::MigrateToLatestXcmVersion<crate::Runtime>,
-	cumulus_pallet_xcmp_queue::migration::v4::MigrationToV4<crate::Runtime>,
-	pallet_identity::migration::versioned::V0ToV1<crate::Runtime, IDENTITY_MIGRATION_KEY_LIMIT>,
-	pallet_uniques::migration::MigrateV0ToV1<crate::Runtime, ()>,
-	runtime_common::migrations::restricted_location::MigrateRestrictedTransferLocation<
-		crate::Runtime,
-		AccountMap,
-	>,
-);
+pub type UpgradeCentrifuge1200 = ();
```

Comment:
Ok, we need to be sure we use the 1103 as RU to apply the previous migrations 👍🏻